File: Python/Daily020_intermediate/AgePlusPlus.py
## AGE PLUS PLUS
##
## challenge #20 (intermediate)
## http://www.reddit.com/r/dailyprogrammer/comments/qnkpp/382012_challenge_20_intermediate/
## 
## 
## sarthak7u@gmail.com
##

def main():
	age = int(raw_input("How old are you? : "))	
	age_months = age * 12
	age_days = age * 365
	age_hours = age_days * 24
	age_minutes = age_hours * 60
	age_seconds = age_minutes * 60

	print "Your age in months : ", age_months
	print "Your age in days : ", age_days
	print "Your age in hours : ", age_hours
	print "Your age in minutes : ", age_minutes
	print "Your age in seconds : ", age_seconds

main()

File: C/Daily020_intermediate/AgePlusPlus.c
/** 
 * AGE PLUS PLUS
 *
 * challenge #20 (intermediate)
 * http://www.reddit.com/r/dailyprogrammer/comments/qnkpp/382012_challenge_20_intermediate/
 *
 * sarthak7u@gmail.com
 */
#include <stdio.h>
#include <stdlib.h>
int main(int argc, char const *argv[])
{
	char age_string[4];
	printf("How old are you? ");
	fgets(age_string,sizeof(age_string), stdin);
	int age = atoi(age_string);

	printf("Months  : %d\n", age*12);
	printf("Days    : %d\n", age*12*30);	
	printf("Hours   : %d\n", age*12*30*24);
	printf("Minutes : %d\n", age*12*30*24*60);
	return 0;
}